__tests__/integration.test.js

```diff
@@ -281,6 +281,31 @@ describe("/api/articles/article:id", () => {
     });
 });
 
+describe("/api/articles/article:id extra query", () => {
+    test("GET:/api/articles/article:id returns the specified article and adds the comment count", () => {
+        return request(app)
+            .get('/api/articles/3')
+            .expect(200)
+            .then(({ body }) => {
+                const { article } = body
+                expect(article).toMatchObject(
+                    {
+                        article_id: expect.any(Number),
+                        title: expect.any(String),
+                        topic: expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        article_img_url: expect.any(String),
+                        comment_count: expect.any(Number),
```

Comment:
No need to test for the other properties as we have already tested for them in a previous test, we can just write a new test checking for the comment_count and both should pass as we've used a flexible matcher like .toMatchObject. This way we are never going back and editing old tests.